apps/help-with-fees/preview/base/kustomization.yaml

```diff
@@ -2,4 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../../base
+  - ../../identity/identity.yaml
```

Comment:
this is missing the patch to add the aat identity file, see:
https://github.com/hmcts/cnp-flux-config/blob/3edbc3f79ea0c839b970a487147ffbeae66bf3ea/apps/rpe/preview/base/kustomization.yaml#L10